extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -142,10 +143,10 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		}
 	}
 
-	async createBook(bookPath: string, contentPath: string): Promise<void> {
-		bookPath = path.normalize(bookPath);
-		contentPath = path.normalize(contentPath);
-		await this.bookTocManager.createBook(bookPath, contentPath);
+	async createBook(): Promise<void> {
+		const bookTocManager = new BookTocManager();
```

Comment:
Do we really want/need to create a new toc manager every time we do this? It looks like we're just using this to create a book so seems like we can just have a single one we re-use